tests/services/test_rdm_service.py

```diff
@@ -246,3 +249,126 @@ def test_draft_w_languages_creation(
         'id': 'eng',
         'title': {'en': 'English', 'da': 'Engelsk'}
     }]
+
+
+#
+# Embargo lift
+#
+@mock.patch('arrow.utcnow')
+def test_embargo_lift_without_draft(
+        mock_arrow, running_app, es_clear, minimal_record, lang):
+    superuser_identity = running_app.superuser_identity
+    service = current_rdm_records.records_service
+    # Add embargo to record
+    minimal_record["access"]["files"] = 'restricted'
+    minimal_record["access"]["status"] = 'embargoed'
+    minimal_record["access"]["embargo"] = dict(
+        active=True, until='2020-06-01', reason=None
+    )
+    # We need to set the current date in the past to pass the validations
+    mock_arrow.return_value = arrow.get(datetime(1954, 9, 29), tz.gettz('UTC'))
+    draft = service.create(superuser_identity, minimal_record)
+    record = service.publish(id_=draft.id, identity=superuser_identity)
+    # Recover current date
+    mock_arrow.return_value = arrow.get(datetime.utcnow())
+
+    service.lift_embargo(_id=record['id'], identity=superuser_identity)
+    record_lifted = service.record_cls.pid.resolve(record['id'])
+
+    assert record_lifted.access.embargo.active is False
+    assert record_lifted.access.protection.files == 'public'
+    assert record_lifted.access.protection.record == 'public'
```

Comment:
question: in line 264 `status` is `embargoed` does this value change after lifiting it? if so I would assert it

Reply:
Yes, nice catch! thanks